src/modules/moderate.ts

```diff
@@ -58,13 +69,17 @@ const bannedWords = [
     "hawk tuah",
     "retard",
     "faggot",
-    "fag", // hey look it's me - koda
+    "fag", // hey look it's us - koda and zander
     "faggots",
     "fags",
     "n*g",
     "n*gg*",
     "n*gg*r",
-    "nigga"
+    "nigga",
+    "grope",
+    "i'm gonna touch you",
+    "im gonna touch you",
```

Comment:
These shouldn't be here. Also too easy to bypass

Reply:
will remove, just thought they might be good to have there